src/exchanges/exchanges-service.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+import fetch from 'node-fetch';
+import Exchange from './exchange';
+
+const airtableBaseUrl = 'https://alpha.selfkey.org/marketplace/i/api/';
+
+export class ExchangesService {
+	static async loadExchangeData() {
+		const response = await fetch(`${airtableBaseUrl}Exchanges`);
+
+		const responseBody = await response.json();
+
+		const exchanges = responseBody.Exchanges.filter(
+			row => row.data && row.data.fields.name
+		).map(row => ({
```

Comment:
The data structure produced here seems a bit weird.

Reply:
how strange? an array of {name: ... , data: ...} I did not modify the schema

Reply:
I guess what I'm wondering is why we're doing this processing on the object in the first place. This is what the input/output (with fake field names because I'm not sure of the real ones) looks like:

```
// Input:

{ name: "foo", field1: "value", field2: "value" }

// Output:

{ name: "foo", data: { name: "foo", field1: "value", field2: "value" } }
```

Surely something like:

```
...
).map(row => row.data.fields)
```

would be more convenient than nesting the same data inside this "data" attribute.

Let's keep it with the existing schema for now. We can refactor later.

Reply:
that's because it goes to sqlite, so data becomes the json field that stores a generic schema